csun-metalab/nomi
# HOTFIX-11-05-2018--profilePageIsBroken

# Description
Fixed split error on profile for gallery view, reorganized the view buttons.
  
# Testing
1. Ensure there is no errors when visiting a student profile in gallery and list view.
2. Make sure the view selector works properly with no errors.
